src/lib/Repository/Values/Content/ContentType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType as APIContentType;
+
+/**
+ * Adds a new method to content type that allows us to provide extra functionality.
+ */
+class ContentType extends \eZ\Publish\Core\Repository\Values\ContentType\ContentType
+{
+    public function __construct(APIContentType $contentType)
```

Comment:
probably worth adding in additional properties for future extension/consistenct with others.